### PR TITLE
fix(infra): fix import

### PR DESCRIPTION
**Pull request by Orta21**
Ref. 726

Ref: 726

### Description

Lambda import

### Release Plan

- [ ] Asap
- [ ] _[action 1]_
- [ ] _[action 2]_

### CODE REVIEW

File: packages/packages/api/src/medical/client/metriport.ts [hunk 1]
```diff
@@ -354,16 +354,18 @@ export class MetriportMedicalApi {
   }
 
   /**
-   * Returns a URL that can be used to download the document.
+   * Fetches the document from S3 and sends a presigned URL
```
Comment (leite08) [inline packages/packages/api/src/medical/client/metriport.ts:357]: I don't think we should update the description, is essentially what we did before, just with a diff format, right? No need to expose to the client that we need to download from S3

File: packages/packages/api/src/medical/client/metriport.ts [hunk 2]
```diff
@@ -354,16 +354,18 @@ export class MetriportMedicalApi {
...
    *
-   * @param fileName The file name of the document in s3.
-   * @return presigned url that expires in 20 seconds
+   * @param req.query.fileName The file name of the document in s3.
+   * @param req.query.conversionType The doc type to convert to.
```
Comment (leite08) [inline packages/packages/api/src/medical/client/metriport.ts:360]: We should list the valid values, ideally also making a union type on `conversionType` (`pdf` | `xml`).

File: packages/packages/api/src/medical/client/metriport.ts (hunk 1 above)
Comment (Orta21) [reply packages/packages/api/src/medical/client/metriport.ts]: Updated

File: packages/packages/api/src/medical/client/metriport.ts
Comment (leite08) [inline packages/packages/api/src/medical/client/metriport.ts:1]: need to deploy to NPM and update places that use this package, right?

Comment (Orta21) [reply packages/packages/api/src/medical/client/metriport.ts:1]: Yea still need to deploy the package dont know how we want to go about doing that if i merge this to master

Comment (Goncharo) [reply packages/packages/api/src/medical/client/metriport.ts:1]: we could just release this package in a new version, and keep the old one in other places, right?

File: packages/packages/api/src/medical/client/metriport.ts [hunk 3]
```diff
@@ -356,14 +356,16 @@ export class MetriportMedicalApi {
...
    *
-   * @param fileName The file name of the document in s3.
-   * @return presigned url that expires in 20 seconds
+   * @param req.query.fileName The file name of the document in s3.
+   * @param req.query.conversionType The doc type to convert to. Valid values are "html" and "pdf".
+   * @return presigned url
    */
   //eslint-disable-next-line @typescript-eslint/no-explicit-any
-  async getDocumentUrl(fileName: string): Promise<{ url: string }> {
+  async getDocumentUrl(fileName: string, conversionType: string): Promise<{ url: string }> {
```
Comment (Goncharo) [inline packages/packages/api/src/medical/client/metriport.ts:364]: conversionType should be optional

**Review by Goncharo — CHANGES_REQUESTED** [review]

File: packages/packages/api/src/medical/client/metriport.ts (hunk 3 above)
Comment (Goncharo) [reply packages/packages/api/src/medical/client/metriport.ts]: also might as well make it a type literal and/or enum (ie html/pdf)

File: packages/packages/api/src/medical/client/metriport.ts
Comment (leite08) [reply packages/packages/api/src/medical/client/metriport.ts:1]: I think just release a new, non-alpha version and update references

Comment (leite08) [reply packages/packages/api/src/medical/client/metriport.ts:1]: Also, remember to back merge into develop so we don't have conflicts afterwards

File: packages/packages/api/src/medical/client/metriport.ts (hunk 3 above)
Comment (leite08) [reply packages/packages/api/src/medical/client/metriport.ts]: [union types](https://github.com/metriport/metriport/pull/407#discussion_r1212515214) 🙏

**Review by Goncharo — APPROVED** [review]
:shipit:

**Pull request merged**